user/drawterm/template.py

```diff
@@ -0,0 +1,44 @@
+pkgname = "drawterm"
+_commit = "f11139d4c918802a87730bc14d094670ee4ce572"
+pkgver = "0_git20240703"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_env = {"CONF": "linux"}
+hostmakedepends = [
+    "gmake",
+    "pkgconf",
+    "wayland-progs",
+]
+makedepends = [
+    "libxkbcommon-devel",
+    "linux-headers",
+    "pipewire-devel",
+    "wayland-devel",
+    "wayland-protocols",
+]
+pkgdesc = "Interface to Plan 9 systems"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "MIT"
+url = "https://drawterm.9front.org"
+source = [
+    f"https://git.9front.org/git/plan9front/drawterm/{_commit}/snap.tar.gz",
+    "!https://gitlab.freedesktop.org/wlroots/wlr-protocols/-/raw/2b8d43325b7012cc3f9b55c08d26e50e42beac7d/unstable/wlr-virtual-pointer-unstable-v1.xml",
```

Comment:
you have to rename the file (`>newname` on the end, etc) so it changes name on update

i guess if we're pulling it like this anyway then may as well actually package it since it's less effort (regardless what i said before)

Reply:
the name of the downloaded file is `wlr-virtual-pointer-unstable-v1.xml`, so if you change the entire hash in front of it to download some hypothetical different file it would still reuse the cached one

(for this specific case this is more an imaginary problem, but in general)